### ISSUE TITLE
C0321 with a print

### ISSUE THREAD

**Issue by henri9813** [PyCQA/pylint#2696]
Hello :) 

Thanks this post https://stackoverflow.com/a/22413950

We know that in python 3, print is a `function` not an `statement`

### Steps to reproduce
```
if arguments.verbose: print("Cloning template into: " + arguments.path)
```

### Current behavior
```
my_project/__main__.py:29:30: C0321: More than one statement on a single line (multiple-statements)
``` 

### Expected behavior
Nothing, in python 3 it's a function, not a statement.

### pylint --version output
pylint 2.2.2
astroid 2.1.0
Python 3.6.7 (default, Oct 22 2018, 11:32:17) 
[GCC 8.2.0]

Best regards

**Comment by PCManticore** [comment]
Thanks for the issue, but the check here is correct because you have two logical statements on the same line, which is what the message is actually saying

**Comment by henri9813** [comment]
Thanks for your fast answer, but i don't see the second logical statement ? 

May the print is consider as a logical statement in python3 ?

Best regards

**Comment by henri9813** [comment]
@PCManticore

**Comment by dmtucker** [comment]
@henri9813 I suspect there is nothing special about `print` here. The 2 logical statements are `if` and a function invocation. For example, this also triggers `multiple-statements`:
``` python
if True: object()
```

**Comment by henri9813** [comment]
Thanks @dmtucker,
So we consider a function execution as a `logiciel statement` ? 

Regard

**Comment by dmtucker** [comment]
@henri9813 Yes. Consider that a statement _must_ go there. If it weren't a function invocation, it could be an assignment or a `pass` (or `with`, `if`, `while`, etc.) statement.

Also note the documentation: https://docs.python.org/3/reference/compound_stmts.html#if
```
"if" expression ":" suite
```
where
```
suite         ::=  stmt_list NEWLINE | NEWLINE INDENT statement+ DEDENT
```

**Comment by henri9813** [comment]
Oh okay!
Thank you @dmtucker  ! :+1:
